memgpt/client/client.py

```diff
@@ -423,7 +423,7 @@ def send_message(
     ) -> MemGPTResponse:
         messages = [MessageCreate(role=role, text=message, name=name)]
         # TODO: figure out how to handle stream_steps and stream_tokens
-        request = MemGPTRequest(messages=messages, stream_steps=stream)
+        request = MemGPTRequest(messages=messages, stream_steps=stream, return_message_object=True)
```

Comment:
@sarahwooders `= True` means the type that comes back is `Message`. `= False` means the type that comes back is `InnerThoughts` / `FunctionCall` / ... (these are now typed too, vs previously they were dicts)